api/types/plugin.go

```diff
@@ -75,6 +75,8 @@ const (
 	PluginTypeSCIM = "scim"
 	// PluginTypeDatadog indicates the Datadog Incident Management plugin
 	PluginTypeDatadog = "datadog"
+	// PluginTypeAWSIC indicates the AWS IdentityCenter integration
+	PluginTypeAWSIC = "aws-ic"
```

Comment:
I will need full name "aws-identity-center" for readability since the value will be used in user facing systems, web UI, yaml config for oidc integration etc. 
```suggestion
	PluginTypeAWSIdentityCenter = "aws-identity-center"
```